presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/TypeConverter.java

```diff
@@ -176,6 +177,9 @@ public static Type toPrestoType(org.apache.iceberg.types.Type type, TypeManager
         if (type instanceof MapType) {
             return fromMap((MapType) type);
         }
+        if (type instanceof TimeType) {
```

Comment:
This should only allow `time(6)`. We should probably add a `TIME_MICROS` constant for `createTimeType(6)` like I did in the timestamps PR. Then add another check
```java
if (type instanceof TimestampType) {
    throw new PrestoException(NOT_SUPPORTED, format("Time precision (%s) not supported for Iceberg. Use \"time(6)\" instead.", ((TimeType) type).getPrecision()));
}

Reply:
Done.